packages/hotspot-utils/src/hotspotUtils.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  AccountMeta,
+  Connection,
+  PublicKey,
+  TransactionInstruction,
+  TransactionMessage,
+  VersionedMessage,
+  VersionedTransaction,
+} from "@solana/web3.js";
+import { AnchorProvider, Program, Wallet } from "@project-serum/anchor";
+import { HeliumEntityManager } from "@helium/idls/lib/types/helium_entity_manager";
+import {
+  rewardableEntityConfigKey,
+  iotInfoKey,
+  mobileInfoKey,
+  init,
+} from "@helium/helium-entity-manager-sdk";
+import { subDaoKey } from "@helium/helium-sub-daos-sdk";
+import { Asset, AssetProof, getAssetProof } from "./mplAssetAPI";
+import {
+  PROGRAM_ID as BUBBLEGUM_PROGRAM_ID,
+  createTransferInstruction,
+} from "@metaplex-foundation/mpl-bubblegum";
+import {
+  ConcurrentMerkleTreeAccount,
+  SPL_ACCOUNT_COMPRESSION_PROGRAM_ID,
+  SPL_NOOP_PROGRAM_ID,
+} from "@solana/spl-account-compression";
+
+export type SolHotspot = Awaited<ReturnType<typeof getSolHotspotInfo>>;
+
```

Comment:
We should add a function that does search assets so that we can get just the helium NFTs:

```bash
curl --request POST \
     --url https://rpc-devnet.aws.metaplex.com/ \
     --header 'Content-Type: application/json' \
     --data '{
      "jsonrpc": "2.0",
      "method": "search_assets",
      "id": "get-assets-op-1",
      "params": {
        "ownerAddress": "BcJzP2hEYgzjUwpHEtS6RhuqGfEJVx8Rq3MejujAAWrR",
        "page": 0,
        "creatorAddress": "9XtEZRnuWizL8vGoNwRX636guuSXTjQvsf3wAXmH4C1b",
        "creatorVerified": true,
        "sortBy": {"sortBy": "created", "sortDirection": "asc"}
      }
    }'

```

Where you get creatorAddress from 

```js
import { entityCreatorKey } from "@helium/helium-entity-manager-sdk";

const hntMint = new PublicKey("...hnt mint...")
const creator = entityCreatorKey(hntMint)[0]
```